### DISCUSSION TITLE
Curtain and Light combined switch device

### DISCUSSION THREAD

**Discussion by NashHome** [make-all/tuya-local#714] [General]
Hi,
I configured my Curtain and 2 Lights Switch AVATTO Tuya WIFI.
It recognized the curtain only. I've found in the supported devices list Avatto curtain and light switch which was not shown up as an option to pick up during the configuration. It added 2 entities for the curtain (the cover and the backlight) but not the switches.

Please advise:
1. How can I manually set the device entities to be attributed to specific YAML file?
2. I'd like to duplicate the /config/custom_components/tuya_local/devices/avatto_curtain_light.yaml to adjust it for supporting 2 lights and to point the device to this file. How it should be done?
The light DP codes are 101 and 102 Boolean.

**Comment by make-all** [comment]
You are asking the wrong questions here.  You have already assumed a solution, but it is the wrong one.

Your starting point is that you have a new device that is not supported.
The next step is to gather information from the logs and Tuya portal about that device.  Do not skip this step by assuming that another device should be compatible because it is similar or made by the same manufacturer.  In the Tuya ecosystem, this is seldom the case.
